app/views/posts/_posts_display.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="users-posts">
+<% @user.posts.each do |post| %>
+  <div class="post">
+    <%= link_to post.Title, user_post_path(user_id: post.author_id, id: post.id), class: 'custom-link-s' %>
+    <p><%= post.Text %></p>
+    <%= render_text_section(post) %>
+  </div>
+
+  <%= render_comments_section(post) %>
+<% end %>
```

Comment:
- [ ] As I mentioned before, please use the `recent_comments` method to display the most 5 recent comments inside `post index view` 

https://github.com/danielamoreno699/blog-app-rails/blob/4c6e426fb398f0c76be30066a1a714590f37a48b/app/helpers/posts_helper.rb#L17-L18

Hint: 
```ruby
post.recent_comments.each 
```